laws/views.py

```diff
@@ -873,6 +908,30 @@ def knesset_proposal_auto_complete(request):
 
     return HttpResponse(json.dumps(result), mimetype='application/json')
 
+@require_http_methods(["GET"])
+def committee_meeting_auto_complete(request):
+    if not request.GET.get('query'):
+        raise Http404
+
+    q = request.GET['query']
+
+    options = CommitteeMeeting.objects.filter(
+        Q(date_string__icontains=q) | Q(topics__icontains=q)
+    )[:30]
+
+    data = []
+    suggestions = []
+    for i in options:
+        title = u'{0} - {1}'.format(i.date_string, i.topics)
+        data.append(i.id)
+        suggestions.append(title)
+
+    result = {'query': request.GET['query'],
+              'suggestions': suggestions,
+              'data': data}
+
+    return HttpResponse(json.dumps(result), mimetype='application/json')
```

Comment:
unfortunately not in v1.6 ..

should be easy to backport it but that's a different issue I guess